.github/labeler.yml

```diff
@@ -0,0 +1,10 @@
+documentation:
+- docs/**/*
```

Comment:
Should all the markdown files outside the docs folder be added?

Reply:
Yes, I thought it was necessary, so I modified it.